base-template/base-template.md

```diff
@@ -1,77 +1,123 @@
+{Template author tip:
 
-# Base template
+* This base template captures the structure and common elements that all doctype templates should follow. It provides guidance for template authors to follow when creating a new template.
+* “Template author and template reviewer tips” should be removed from the final {doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
```

Comment:
```suggestion
* Adjust other tips to align with the doctype you are writing for.
```
or "modify".
"Tweak" has a drug connotation in the USA.

Reply:
Thanks updated as suggested.